.travis.yml

```diff
@@ -52,18 +52,18 @@ matrix:
         - NUMPYSPEC="numpy==1.15.4"
     # Test SciPy with the debug version of Python.
     # However travis only specifies regular or development builds of Python
-    # so we must install python3.5-dbg using apt.
+    # so we must install python3.7-dbg using apt.
     - language: generic
       env:
-        - USE_DEBUG=python3.5-dbg
+        - USE_DEBUG=python3.7-dbg
         - TESTMODE=fast
         - NUMPYSPEC="--upgrade numpy"
       addons:
         apt:
           packages:
             - *common_packages
-            - python3.5-dbg
-            - python3.5-dev
+            - python3.7-dbg
+            - python3.7-dev
```

Comment:
Looks like Travis can't find these package names---maybe they are slightly different now or we need a higher version base image of the distro?